src/EdFi.AnalyticsMiddleTier.Tests/TestCases/AssessmentFact/MP_2013_Mathematics_Seventh_grade_should_have_ReportingMethod.xml

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<TestCase>
+  <DBMS>Any</DBMS>
+  <ControlDataInsertion>
+  </ControlDataInsertion>
+  <Query>
+      SELECT ReportingMethod
+      FROM analytics.asmt_AssessmentFact
+      WHERE ObjectiveAssessmentKey = 'MP-2013-Mathematics-Seventh grade-Seventh grade Mathematics-38-uri://ed-fi.org/Assessment/AssessmentTestCase.xml';
+  </Query>
+  <Result>
+      <ReportingMethod></ReportingMethod>
```

Comment:
I might not understand the test fully, but should this be filled in with a value instead of empty, since it is for testing that a reporting method is returned?

Reply:
We added the LEFT JOIN, then ReportingMethod could be null. So, I added a COALESCE expression and I need a testcase to test it. The test case returns Empty, but the name could be confusing. I renamed the file to include the _Empty suffix.